integration-cli/docker_cli_network_unix_test.go

```diff
@@ -135,6 +136,11 @@ func (s *DockerNetworkSuite) TestDockerNetworkCreateDelete(c *check.C) {
 	assertNwNotAvailable(c, "test")
 }
 
+func (s *DockerSuite) TestDockerNetworkDeleteNotExists(c *check.C) {
```

Comment:
I'm wondering why this test is needed ? Probably because to validate the refactor still works, but this test should have been add when pushing the `docker network` subcommand, right ? (I ask the question to avoid, or not, conflicts :stuck_out_tongue: )